src/imitation/algorithms/adversarial/airl.py

```diff
@@ -63,32 +63,53 @@ def __init__(
                 "AIRL needs a stochastic policy to compute the discriminator output.",
             )
 
-    def logits_gen_is_high(
+    def logits_expert_is_high(
         self,
         state: th.Tensor,
         action: th.Tensor,
         next_state: th.Tensor,
         done: th.Tensor,
         log_policy_act_prob: th.Tensor,
     ) -> th.Tensor:
-        """Compute the discriminator's logits for each state-action sample."""
+        r"""Compute the discriminator's logits for each state-action sample.
+
+        In Fu's AIRL paper (https://arxiv.org/pdf/1710.11248.pdf), the
+        discriminator output was given as
+        $$
+        D_{\theta}(s,a) =
+        \frac{ \exp{r_{\theta}(s,a)} } { \exp{r_{\theta}(s,a)} + \pi(a|s) }
+        $$
+        with a high value corresponding to the expert and a low value corresponding to
+        the generator.
+
+        In other words, the discriminator output is the probability that the action is
+        taken by the expert rather than the generator.
+
+        The logit of the above is given as
+        $$
+        \operatorname{logit}(D_{\theta}(s,a)) = r_{\theta}(s,a) - \log{ \pi(a|s) }
+        $$
+        which is what is returned by this function.
+
+        Args:
+            state: ?
```

Comment:
It looks like you were using `?` as a placeholder here and forgot to fill it in? In the analogous method for GAIL, we don't have an `Args` section in the docstring at all. For completeness it would probably be best to include full `Args` sections in both docstrings. At least it should be consistent for the two methods.